cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -270,6 +270,21 @@ UAF takes advantage of existing security technologies present on devices for aut
 
 U2F augments password-based authentication using a hardware token (typically USB) that stores cryptographic authentication keys and uses them for signing. The user can use the same token as a second factor for multiple applications. U2F works with web applications. It provides **protection against phishing** by using the URL of the website to lookup the stored authentication key.
 
+## Autonomous TOTP
+
+The Time-based One-Time Password algorithm (TOTP) is an extension of the HMAC-based One-time Password algorithm (HOTP) generating a one-time password by instead taking uniqueness from the current time. It has been adopted as Internet Engineering Task Force[1] standard RFC 6238, is the cornerstone of Initiative For Open Authentication (OATH), and is used in a number of [two-factor authentication](https://en.wikipedia.org/wiki/Two-factor_authentication) systems.
```

Comment:
This is all correct, but seems to be a bit to dry and technical for the cheat sheet. It's good to have a link where readers can learn more about the technical details of TOTP, but this doesn't seem like stuff that everyone needs to know.